book-library/index.html

```diff
@@ -61,12 +64,13 @@ <h1>Library</h1>
             value=""
           />Read
         </label>
-        <input
-          type="submit"
-          value="Submit"
+        <button
+          type="button" <!-- Changed input type to button -->
```

Comment:
Good work. Could you tell me why having type set to submit in this context causes problems?